signer/client/signer_trust.go

```diff
@@ -180,16 +179,7 @@ func (trust *NotarySigner) ListAllKeys() map[string]string {
 // CheckHealth checks the health of one of the clients, since both clients run
 // from the same GRPC server.
 func (trust *NotarySigner) CheckHealth(timeout time.Duration) error {
-
-	// Do not bother starting checking at all if the connection is broken.
-	if trust.clientConn.State() != grpc.Idle &&
```

Comment:
You are right @riyazdf , ConnectivityState is still there but it can not directly accessed by `ClientConn` any more. It was now hold by another structure `addrConn` which I would like to use when doing the refactor of the health check mechanism as I said below.